### PR TITLE
updating Example script to include telescope

### PR DESCRIPTION
**Pull request by maxecharles**

### CODE REVIEW

**Comment by maxecharles** [comment]
```python
# imports
import dLuxToliman as dlT
from matplotlib import pyplot as plt

osys = dlT.TolimanOpticalSystem(psf_npixels=128)  # creating Toliman optical system
source = dlT.AlphaCen(n_wavels=3, separation=8, position_angle=30)  # creating Alpha Centauri source
tel = dlT.Toliman(source=source, osys=osys)  # creating Toliman telescope

psf = tel.model()  # running optical simulation

# plotting
plt.imshow(psf ** .5, cmap='inferno')
plt.title(r'Toliman $\sqrt{PSF}$')
plt.show()
```

File: README.md [hunk 1]
```diff
@@ -31,13 +31,15 @@ A very basic example of how to use this package is as follows:
...
 source = dlT.AlphaCen(n_wavels=3, separation=8, position_angle=30)  # creating Alpha Centauri source
-psf = osys.model(source)  # running optical simulation
+tel = dlT.Toliman(source=source, osys=osys)  # creating Toliman telescope
+
+psf = tel.model()  # running optical simulation
 
 # plotting
 plt.imshow(psf ** .5, cmap='inferno')
-plt.title('Toliman $\sqrt{PSF}$')
+plt.title(r'Toliman $\sqrt{PSF}$')
```
Comment (CLangford2098) [inline README.md:44]: I personally think that `plt.title(r'Toliman $\sqrt{\text{PSF}}$')` looks better

File: .gitignore [hunk 2]
```diff
@@ -5,3 +5,5 @@ dist/
 build/
 dLuxToliman.egg-info/
 .idea/
+.vscode/
+.gitignore
```
Comment (CLangford2098) [inline .gitignore]: Is this so everyone can maintain their own gitignore?

File: README.md (hunk 1 above)
Comment (maxecharles) [reply README.md:44]: yeah it does but shot not regenerating the image

File: .gitignore (hunk 2 above)
Comment (maxecharles) [reply .gitignore]: ye

File: README.md (hunk 1 above)
Comment (CLangford2098) [reply README.md:44]: That's Max!

File: .gitignore (hunk 2 above)
Comment (CLangford2098) [reply .gitignore]: No worries

File: README.md (hunk 1 above)
Comment (CLangford2098) [reply README.md:44]: Maybe add this for future PRs

**Review by CLangford2098 — APPROVED** [review]

**Pull request merged**